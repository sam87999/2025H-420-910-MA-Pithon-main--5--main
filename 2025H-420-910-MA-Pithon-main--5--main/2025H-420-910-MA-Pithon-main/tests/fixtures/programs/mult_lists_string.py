# Test de Alexandrine Dub√©
a = "Hello"	
b = 5
print(a * b) # Output: HelloHelloHelloHelloHello

c = [21, "hey", True]	
d = 3
print(c * d) # Output: [21, 'hey', True, 21, 'hey', True, 21, 'hey', True]

e = (23, "oki", False, None)
f = 3
print(e * f) # Output: (23, 'oki', False, None, 23, 'oki', False, None, 23, 'oki', False, None)

