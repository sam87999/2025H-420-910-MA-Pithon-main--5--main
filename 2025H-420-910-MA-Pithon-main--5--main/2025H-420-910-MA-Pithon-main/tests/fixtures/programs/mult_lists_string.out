HelloHelloHelloHelloHello
[21, 'hey', True, 21, 'hey', True, 21, 'hey', True]
(23, 'oki', False, None, 23, 'oki', False, None, 23, 'oki', False, None)
